services/nexus/nexus.tester.js

```diff
@@ -18,32 +18,36 @@ function mockNexusCreds() {
 }
 
 t.create('live: search release version valid artifact')
-  .get('/r/https/repository.jboss.org/nexus/jboss/jboss-client.json')
+  .timeout(15000)
```

Comment:
Now I remember why we were using the jboss instance for these tests.. the sonatype instance is periodically really slow leading to test timeouts.